### ISSUE TITLE
Automatically check Plugin compatibility in CI

### ISSUE THREAD

**Issue by phortx** [vuex-orm/vuex-orm#208]
Hi,

the Pull Request with the pivot tables just brought me to the idea, that we could test if any change breaks plugins in the CI build of the vuex-orm core.

There could be a list of relevant plugins and the CI script clones each of the plugins in a dedicated directory, runs yarn/npm install, build and links the new version of the core in the respective plugin and runs the test suite of the plugin. 

That way the CI build breaks when a change affects one of the more complicated plugins.

What do you think?

**Comment by kiaking** [comment]
Yeah! this is quite nice idea. It would be really handy if this goes live.
